app/services/notifications.py

```
@@ -97,12 +101,15 @@ def send_ntfy(topic, title, message, priority='default'):
             data = message.encode('utf-8')
-            req = Request(url, data=data, headers={
+            headers = {
                 'Title': title,
                 'Priority': priority,
                 'Tags': 'car',
                 'User-Agent': 'May-Vehicle-Manager/1.0',
-            })
+            }
+            if token:
+                headers['Authorization'] = f'Bearer {token}'
+            req = Request(url, data=data, headers=headers)
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)